src/redux/store.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import booksSliceReducer from './books/booksSlice';
+
+const store = configureStore({
+  reducer: {
+    book: booksSliceReducer,
+  },
+});
+
+export default store;
```

Comment:
- [ ] Nice work in this `store` section 👏 . However, you need both `books` and `categories` to be added to the `configurestore` method, as it was arranged in the duck pattern. This was made so to help you follow best practices 👍 

## Duck pattern:
![image](https://user-images.githubusercontent.com/50990925/229974572-13891f77-1217-4309-b527-cd0326f873af.png)

- [ ] Your store should like this:
```

import { configureStore } from "@reduxjs/toolkit"
import booksReducer from "./books/booksSlice"
import categoriesReducer from "./categories/categoriesSlice"

const store = configureStore({
  reducer: { books: booksReducer, categories: categoriesReducer },
})

export default store
```
- You can implement in your own way, but the outcome should be in the format above 👍